src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/ReasonsForRecallServiceTest.kt

```diff
@@ -57,14 +66,15 @@ class ReasonsForRecallServiceTest {
     val dossierContext = mockk<DossierContext>()
     val recallId = ::RecallId.random()
     val expectedBytes = "Some expected content".toByteArray()
+    val createdByUserId = ::UserId.random()
 
     every { documentService.getLatestVersionedDocumentContentWithCategoryIfExists(any(), DOSSIER) } returns expectedBytes
     every { dossierContext.recall } returns Recall(recallId, randomNoms(), ::UserId.random(), OffsetDateTime.now(), FirstName("Barrie"), null, LastName("Badger"))
 
-    val generatedPdf = underTest.getDocument(dossierContext).block()!!
+    val generatedPdf = underTest.getPdf(dossierContext, createdByUserId).block()!!
 
     assertArrayEquals(expectedBytes, generatedPdf)
-    verify(exactly = 0) { documentService.storeDocument(any(), any(), any(), any()) }
+    verify(exactly = 0) { documentService.storeDocument(any(), createdByUserId, any(), any(), any()) }
```

Comment:
Tiny but I guess `createdByUserId` here could be another `any()`?

Reply:
Will add to next PR :)